src/Components/Blazor/Blazor/src/Hosting/WebAssemblyHostBuilder.cs

```diff
@@ -106,6 +106,8 @@ private void CreateServiceProvider()
             });
 
             // Needed for authorization
+            // However, since authorization isn't on by default, we could consider removing these and
+            // having a separate services.AddBlazorAuthorization() call that brings in the required services.
```

Comment:
It's client-side Blazor so I'm not dealing with it now, but if we do this, we could drop the `M.E.Options` dependency from default client-side Blazor projects.